Tools/cases_generator/generate_cases.py

```diff
@@ -14,23 +14,76 @@
 
 import parser
 
-DEFAULT_INPUT = "Python/bytecodes.c"
-DEFAULT_OUTPUT = "Python/generated_cases.c.h"
+DEFAULT_INPUT = os.path.relpath(
+    os.path.join(os.path.dirname(__file__), "../../Python/bytecodes.c")
+)
+DEFAULT_OUTPUT = os.path.relpath(
+    os.path.join(os.path.dirname(__file__), "../../Python/generated_cases.c.h")
+)
```

Comment:
Sorta defeats the purpose of `os.path.join`... ;)

```suggestion
DEFAULT_INPUT = os.path.relpath(
    os.path.join(os.path.dirname(__file__), os.pardir, os.pardir, "Python", "bytecodes.c")
)
DEFAULT_OUTPUT = os.path.relpath(
    os.path.join(os.path.dirname(__file__), os.pardir, os.pardir, "Python", "generated_cases.c.h")
)
```